appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -44,6 +44,9 @@ export class AppInsightsCreateStep extends AzureWizardExecuteStep<IAppServiceWiz
                     wizardContext.appInsightsComponent = await client.components.createOrUpdate(rgName, aiName, { kind: 'web', applicationType: 'web', location: appInsightsLocation });
                     const createdNewAppInsights: string = localize('createdNewAppInsights', 'Successfully created Application Insights resource "{0}".', aiName);
                     ext.outputChannel.appendLog(createdNewAppInsights);
+                } else if (pError.errorType === 'AuthorizationFailed') {
+                    const createAppInsightsNotAvailable: string = localize('appInsightsNotAvailable', 'Skipping creating an Application Insights because the user doesn\'t have the right permissions.');
```

Comment:
For resource groups and plans we let them select an existing one - how come we're not doing that here?

Reply:
We can do that here as well. I think we were going with this as it might cause too many prompts and we do have an option of creating without the AppInsights. 
Do you think it is better to prompt to select the existing?

Reply:
How many prompts is it? I could maybe see skipping it for basic create, but for advanced create it seems weird to silently skip it when they explicitly tried to create one

Reply:
I guess with Web Plan Contributor, there is only one prompt and that is with selecting existing resource groups. 
There shouldn't be an error with creating an App Service Plan. 

I agree I think we should skip it for basic create and prompt for advanced create.